its/src/test/java/com/sonar/it/scanner/msbuild/TestUtils.java

```diff
@@ -212,8 +213,18 @@ public static void runMSBuildWithBuildWrapper(Orchestrator orch, Path projectDir
   }
 
   public static void runMSBuild(Orchestrator orch, Path projectDir, String... arguments) {
-    BuildResult r = runMSBuildQuietly(orch, projectDir, arguments);
-    assertThat(r.isSuccess()).isTrue();
+    int attempts = 0;
```

Comment:
I am not uber happy with this approach, however the alternative is to try out the `/nr:false` approach, monitor, see if it affects the tests time etc

however this will just make things quiet and it's not that ugly either

Reply:
(I'm not sure if I'm being pragmatic or lazy here, please share opinions)

Reply:
I agree. We should have red jobs when there's really an issue. :/
What if you add this logic into `runMSBuildQuietly` so that it can be re-used.

Reply:
initially I put it here because the private method was only used here but you're right